src/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -319,6 +328,16 @@ public partial class UnhandledExceptionEventArgs : System.Threading.ThreadExcept
         public UnhandledExceptionEventArgs(bool exitApplication, System.Exception exception) : base (default(System.Exception)) { }
         public bool ExitApplication { get { throw null; } set { } }
     }
+    public partial class User
+    {
+        public User() { }
+        public string Name { get { throw null; } }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Advanced)]
+        public System.Security.Principal.IPrincipal CurrentPrincipal { get { throw null; } set { } }
+        public bool IsAuthenticated { get { throw null; } }
+        protected virtual System.Security.Principal.IPrincipal InternalPrincipal { get { throw null; } set { } }
```

Comment:
Nit, it's helpful to sort the members in the ref sources because periodically we regenerate them mostly automatically and then they are sorted, so it causes more diff.